dev-python/aiodns/files/aiodns-2.0.0-fix-typing-dependency.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index a19d32f..ed86e49 100644
+--- a/setup.py
++++ b/setup.py
+@@ -18,7 +18,7 @@ setup(name             = "aiodns",
+       url              = "http://github.com/saghul/aiodns",
+       description      = "Simple DNS resolver for asyncio",
+       long_description = codecs.open("README.rst", encoding="utf-8").read(),
+-      install_requires = ['pycares>=3.0.0', 'typing; python_version<"3.7"'],
++      install_requires = ['pycares>=3.0.0', 'typing; python_version<"3.5"'],
```

Comment:
I think the dependency is correct, and the ebuild should be fixed to have a dependency like the aiohttp ebuild:
```
$(python_gen_cond_dep 'dev-python/typing-extensions[${PYTHON_USEDEP}]' python3_{5,6})
```

Reply:
Actually, you're right. I see that the required typing stuff is here:

https://docs.python.org/3.5/library/typing.html

aiodns only uses these:
```python
from typing import (
    Any,
    List,
    Optional,
)
```